ComponentKit/Utilities/CKComponentActionInternal.h

```diff
@@ -57,11 +61,11 @@ class CKTypedComponentActionBase {
 
   bool operator==(const CKTypedComponentActionBase& rhs) const;
 
-private:
   // Destroying this field calls objc_destroyWeak. Since this is the only field
   // that runs code on destruction, making this field the first field of this
   // object saves an offset calculation instruction in the destructor.
   __weak id _targetOrScopeHandle;
+  dispatch_block_t _block;
```

Comment:
Yeah, you're right.

Would it be out of the question to convert CKComponentAction to a NSObject (not in this diff, just, in general)